ipv6calc/test_ipv6calc.sh

```diff
@@ -252,6 +265,9 @@ cat <<END | grep -v '^#'
 1.2.3.r4									ipv4addr
 2.2.3.4/-1									ipv4addr
 2.2.3.4/33									ipv4addr
+1.2.3.4/									ipv4addr
+/10										ipv4addr
```

Comment:
do your really want to catch /10 as IPv4 address?  - in principle it can be also an IPv6 address...

Reply:
`/10` can't be IPv6 address, because any IPv6 address will have at least two colons.

But now I've re-read some info, and realized that while `1.2/32` (CIDR notation) should stand for `1.2.0.0/32`, `1.2` stands for classful notation, and should be interpeted as `1.0.0.2`. And there could be even more hilarous `1.65535`, which should be translated to `1.0.255.255`... I think that the last one is not used anywhere and propose keep treating such as error. But  CIDR and classful notation must be distingushed. Not sure how to interpret `--forceprefix` and `--noprefix-length`, though.

Reply:
Or maybe we should forget about classful routing in XXI century? Nobody does it nowadays, AFAIK.

Reply:
"1.2" and "1.65535" should be not supporte imho. And also I don't like the idea of supporting "/10" (never seen so far) - do you have a real-world example, where it would be useful or even displayed? Because it would be the shortcut of "0.0.0.0/10" somehow, also writable as "0/10".

Reply:
I do not support `/10`, it's in `testscenarios_auto_bad()`. And looking at the test code, the `ipv4addr` "result" is ignored anyway.